scripts/entrypoint.sh

```diff
@@ -13,6 +13,7 @@ SX_THREADS=${SX_THREADS:-1}
 THREAD_SGI_PRIO=${THREAD_SGI_PRIO:-80}
 SGI_THREADS=${SGI_THREADS:-8}
 BYPASS_UL_PFCP_RULES=${BYPASS_UL_PFCP_RULES:-no}
+GTP_EXTENSION_HEADER_PRESENT=${GTP_EXTENSION_HEADER_PRESENT:-yes}
```

Comment:
Rohan (@orion-belt )

As mentioned today in my discussion, I would like this to `off` by default for 4G usage